Formula/glbinding.rb

```diff
@@ -11,12 +11,31 @@ class Glbinding < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "homebrew/versions/glfw3" => :optional
   needs :cxx11
 
   def install
+    args = std_cmake_args
+
+    if build.with? "glfw3"
+      args << "-DOPTION_BUILD_TOOLS=ON"
+    else
+      args << "-DOPTION_BUILD_TOOLS=OFF"
+    end
+
     ENV.cxx11
-    system "cmake", ".", *std_cmake_args
-    system "make", "install"
+    system "cmake", ".", *args
+    system "cmake", "--build", ".", "--target", "install"
+  end
+
+  def caveats
+    s = ""
+
+    if build.without? "glfw3"
+      s = "You are building without tool support. To enable tools, reinstall with 'brew reinstall glbinding --with-glfw3'"
```

Comment:
This is slightly wrong. You want `+=` rather than another `=` here, and we should probably wrap it given the length. Similar to how the `node` formula does:

```ruby
    if build.without? "full-icu"
      s += <<-EOS.undent
        Please note by default only English locale support is provided. If you need
        full locale support you should:
          `brew reinstall node --with-full-icu`
      EOS
    end
```